### DISCUSSION TITLE
[Help] Mouse pointer stuck on text selection

### DISCUSSION THREAD

**Discussion by RMcGhee** [Textualize/textual#788] [Q&A] [answered]
Is there a setting I'm missing to keep the mouse pointer as the classic arrow? Whenever the pointer enters the TUI, it changes to the text selection pointer, rather than the classic arrow. I'm on MacOS.

**Comment by willmcgugan** [comment, marked as answer]
I suspect not I'm afraid. It's your terminal emulator which does that, and I don't know of any way of changing that behaviour.

**Reply by RMcGhee** [reply]
Ah, ok. Thanks for the quick response Will, and great work!
